docs/index.js

```diff
@@ -114,7 +114,7 @@
 		tax += (inc - ord[keys[i]]) * num(keys[i]) / 100
 		i--;
 		while (i >= 0) {
-			tax += ord[keys[i + 1]] * num(keys[i]) / 100
+			tax += (ord[keys[i+1]] - ord[keys[i]]) * num(keys[i]) / 100
```

Comment:
just to make sure I understand the change, the fix is taking the amount "over" the previous bracket until we run out of brackets? I think that makes sense, but also, I think this function deserves some unit tests at this point :)

Reply:
Yep, that's correct.

Feel free to write tests, I just noticed the calcs were way off (overestimating ordinary income tax).

Reply:
Real talk: I'm too damn lazy to write tests for this at the moment so I'll just merge it. I was hoping you'd be less lazy than I am, but it seems we're on the same page there. :) Change makes sense!